_search-plugins/star-tree-index.md

```diff
@@ -185,6 +199,48 @@ POST /logs/_search
 
 Using a star-tree index, the result will be retrieved from a single aggregated document as it traverses the `status=500` node, as opposed to scanning through all of the matching documents. This results in lower query latency.
 
+### Date histogram with metric aggregations
+
+You can use [date histograms]({{site.url}}{{site.baseurl}}/aggregations/bucket/date-histogram/) on calendar intervals with metric sub-aggregations.
+
+To use date histogram aggregations and make then searchable in the star-tree index, use the following steps:
+
+- The calendar intervals in a star-tree mapping configuration can have either the request's calendar field or a field of lower granularity than the request field. For example, if an aggregation uses the `month` field, the star-tree search can still use lower granularity fields such as `day`.
+- A metric sub-aggregation must be part of the aggregation request.
+
+#### Example
+
+The following example gets the sum of all the values in the `size` field aggregated for each calendar month, for all error logs with `method:get`:
+
+```json
+POST /logs/_search
+{
+  "query": {
```

Comment:
Is this a valid query ? @sandeshkr419 just double checking. I don't see term/terms etc

Reply:
@bharath-techie: I updated the query to add terms while keeping the method.

Reply:
This is still not a valid query , we don't support range on timestamp. Lets reword this @Naarcha-AWS

Reply:
@bharath-techie: Can you suggest a valid query?

Reply:
```
{
    "size": 0,
    "query": {
        "range": {
            "status": {
                "gte": "200",
                "lte": "400"
            }
        }
    },
    "aggs": {
        "by_hour": {
            "date_histogram": {
                "field": "@timestamp",
                "calendar_interval": "month"
            },
            "aggs": {
                "sum_size": {
                    "sum": {
                        "field": "size"
                    }
                }
            }
        }
    }
}
```

This is a valid query which we can use @Naarcha-AWS